app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -170,6 +294,11 @@ public void onActivityCreated(Bundle savedInstanceState) {
     public void onStart() {
         super.onStart();
         EventBus.getDefault().register(this);
+
+        autoDownload = prefs.getString(PREF_AUTO_DOWNLOAD,STRING_NO_FILTER);
```

Comment:
I think those can be loaded when actually filtering the items. That way, the Fragment does not need to keep preferences as member variables and watch out that they are always updated.